docker/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM ubuntu:20.04
+RUN \
+  # configure the "jhipster" user
+  groupadd jhipster && \
+  useradd jhipster -s /bin/bash -m -g jhipster -G sudo && \
+  echo 'jhipster:jhipster' |chpasswd && \
+  mkdir /home/jhipster/app && \
+  export DEBIAN_FRONTEND=noninteractive && \
+  export TZ=Europe\Paris && \
```

Comment:
Maybe we should use UTC as timezone value?

Reply:
I put it there to avoid any issue related to Timezone . It doesn't affect anything much.